README.md

```diff
@@ -34,6 +34,11 @@ _Getting unreleased versions of `cabal-install`_: gives you a chance to try out
 Currently, we only provide binaries for `x86_64` platforms.
 
 1. _[GitHub preview release built from the tip of the `master` branch](https://github.com/haskell/cabal/releases/tag/cabal-head)_:
```

Comment:
```suggestion
1. _GitHub preview release built from the tip of the `master` branch_: [download from GitHub(https://github.com/haskell/cabal/releases/tag/cabal-head) or use this GHCup command:
```

Reply:
I have added a “to install” after “GHCup command” for clarity.